docs/api/resources/system.rst

```diff
@@ -33,6 +33,8 @@ List resource
       Content-Type: application/json
 
         {
+                "adv_syslogserver": "",
+                "adv_sysloglevel": "info",
```

Comment:
It can't be `info`.
I think testing the changes you greatly improve the quality of your commits.